tox.ini

```diff
@@ -128,8 +124,9 @@ commands =
     double: python -c 'import sys; from astropy import test; test(); sys.exit(test())'
 
 pip_pre =
+    devdeps: true
     predeps: true
-    !predeps: false
+    !devdeps-!predeps: false
```

Comment:
We never chain these two together and my tox-fu is weak, so maybe @saimn , @astrofrog , or @Cadair can review? 🙏

Reply:
This reads as `not devdeps and not predeps`. There's a similar construct already on line 121.

Reply:
(and I just realized that this line is purely redundant so I removed it for clarity)